docsrc/content/Matrix_Vector.fsx

```diff
@@ -13,86 +14,140 @@ open FSharp.Plotly
 <a name="Vector"></a>
 
 ##Vector
+*)
+#r "FSharp.Stats.dll"
+open FSharp.Stats
 
-<a name="Matrix"></a>
+Array.set
 
-##Matrix
+let v = 
+    vector [|2.0; 20.0; 1.|]
 
+(*** include-value:v ***)
 
+let rv = 
+    rowvec [|2.0; 20.0; 1.|]
+
+(*** include-value:rv ***)
+
+(**
+<a name="Matrix"></a>
+
+##Matrix
 
 *)
-//#r "D:/Source/FSharp.Stats/bin/FSharp.Stats.dll"
 #r "FSharp.Stats.dll"
-//open FSharp
-open Microsoft.FSharp.Math
 open FSharp.Stats
-open FSharp.Stats.Algebra
-
 
 
 // http://fdatamining.blogspot.de/2010/03/matrix-and-linear-algebra-in-f-part-i-f.html
-// 1.9.7.8. 
 // http://fdatamining.blogspot.de/search/label/linear%20algebra
 
 let A = 
     matrix [ [ 1.0; 7.0; 2.0 ]
              [ 1.0; 3.0; 1.0 ]
              [ 2.0; 9.0; 1.0 ] ]
 
+(*** include-value:A ***)
```

Comment:
Try to use the matrix pretty printer using: fsi.AddPrinter(FSharp.Stats.FSIPrinter.matrix)